app/services/reports/adult_incontinence_report_service.rb

```diff
@@ -1,98 +1,99 @@
 module Reports
   class AdultIncontinenceReportService
-    ADULT_INCONTINENCE_TYPES = %w[adult_incontinence underpads pads liners].freeze
+    include ActionView::Helpers::NumberHelper
     attr_reader :year, :organization
 
+    # @param year [Integer]
+    # @param organization [Organization]
     def initialize(year:, organization:)
       @year = year
       @organization = organization
     end
 
+    # @return [Hash]
     def report
-      @report ||= {
-        money_spent: money_spent,
-        monthly_adult_incontinence: monthly_adult_incontinence,
-        provided_per_person: provided_per_person,
-        supplies_distributed: supplies_distributed,
-        supplies_purchased: supplies_purchased,
-        supplies_received: supplies_received
-      }
+      @report ||= { name: 'Adult Incontinence',
+                    entries: {
+                      'Adult incontinence supplies distributed' => number_with_delimiter(distributed_supplies),
+                      'Adult incontinence supplies per adult per month' => monthly_supplies&.round || 0,
+                      'Adult incontinence supplies' => types_of_supplies,
+                      '% adult incontinence supplies donated' => "#{percent_donated.round}%",
+                      '% adult incontinence bought' => "#{percent_bought.round}%",
+                      'Money spent purchasing adult incontinence supplies' => number_to_currency(money_spent_on_supplies)
+                    } }
     end
 
-    def columns_for_csv
-      %i[supplies_distributed monthly_adult_incontinence
-         supplies_received supplies_purchased]
+    # @return [Integer]
+    def distributed_supplies
+      @distributed_supplies ||= organization
+                                .distributions
+                                .for_year(year)
+                                .joins(line_items: :item)
+                                .merge(Item.adult_incontinence)
+                                .sum('line_items.quantity')
     end
 
-    def adult_incontinence_items
-      organization.items.where(partner_key: ADULT_INCONTINENCE_TYPES)
+    # @return [Integer]
+    def monthly_supplies
+      # TODO: this is asking "per adult per month" but I'm not sure how that's different
+      # from just getting the average quantity of all line items, which doesn't seem like a useful
+      # metric - and the "per month" seems redundant since it would be the same per adult
+      # per year - we don't keep track of returning adults so every line item is considered
+      # a "new adult"
+      organization
+        .distributions
+        .for_year(year)
+        .joins(line_items: :item)
+        .merge(Item.adult_incontinence)
+        .average('line_items.quantity')
```

Comment:
Will this give the monthly average? Won't think give the average of each distribution? My concern would be if they have unusual distribution cycles like once every 3 months or possibly multiple times in a month?

Reply:
@seanmarcia it's a good question, and I'm not quite sure how to distil all of that into a single number. The fact that they're asking "per person" is odd, because from how I understand the data, each line item is "per person", because we don't keep track of whether we're giving 5 of them to the same person, or if the same person comes back every week. So it doesn't matter whether we track this per month, per year, or any other time frame, it all comes down to the average quantity per line item. Unless I'm misunderstanding something big here.

Reply:
Line items represent the items in a distribution - So the line items in a particular distribution could be 600 size 3 diapers, 400 size 1 diapers, 500 size 5 diapers. Those number could represent 30 kids since most diaper banks give out 50 diapers per child and the total number of diapers in that order was 1500 (1500 / 50 = 30) 

The banks can customize how many of a particular item they give to each person on the item itself -- it is stored in the database in the `distribution_quantity` column.

Reply:
Ah - that was the missing piece of info. So `distribution_quantity` represents how many go to a single individual, and we can use that combined with `quantity` for each line item to get how many individuals received that line item. Then we can use that number across the year to calculate the average.

Reply:
Hmm... actually now I look at this more closely... now isn't this just the average of the `distribution_quantity` for all items that were given out??? We're literally saying "number of products per adult" which seems to be exactly what `distribution_quantity` represents, correct?